### PR TITLE
GEM-46 | Add Get attachment Endpoint

### PR DESCRIPTION
**Pull request by lawos98**

### CODE REVIEW

File: src/main/resources/application-local.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+server:
+  port: 8081
```
Comment (pwgacek) [inline src/main/resources/application-local.yaml]: does it have any purpose or u just forgot to delete this?

File: src/main/kotlin/pl/edu/agh/gem/external/mapper/DefaultAttachmentMapper.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.http.HttpHeaders.CONTENT_LENGTH
+import org.springframework.http.HttpHeaders.CONTENT_TYPE
+import org.springframework.stereotype.Component
+import pl.edu.agh.gem.internal.model.GroupAttachment
+
+@Component
+class DefaultAttachmentMapper : AttachmentMapper {
+    override fun mapToResponseEntity(attachment: GroupAttachment): HttpEntity<ByteArray> {
+        val headers = HttpHeaders()
+        println("Mapping GroupAttachment to ResponseEntity: content type : ${attachment.contentType}")
```
Comment (pwgacek) [inline src/main/kotlin/pl/edu/agh/gem/external/mapper/DefaultAttachmentMapper.kt]: print

**Review by pwgacek — CHANGES_REQUESTED** [review]

File: src/main/kotlin/pl/edu/agh/gem/external/mapper/DefaultAttachmentMapper.kt (hunk 2 above)
Comment (lawos98) [reply src/main/kotlin/pl/edu/agh/gem/external/mapper/DefaultAttachmentMapper.kt]: removed

File: src/main/resources/application-local.yaml (hunk 1 above)
Comment (lawos98) [reply src/main/resources/application-local.yaml]: removed

**Review by pwgacek — APPROVED** [review]

**Pull request merged**